src/styles/style.css

```diff
@@ -0,0 +1,106 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+@import "./stars.css";
+@import "./reset.css";
+
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
```

Comment:
Don't reset margins using * selector. Same with font, you can set the font for the whole page and inherit where needed